command/agent/secure_variable_endpoint_test.go

```diff
@@ -192,7 +194,7 @@ func TestHTTP_SecureVariables(t *testing.T) {
 			require.NoError(t, err)
 			respW := httptest.NewRecorder()
 			obj, err := s.Server.SecureVariableSpecificRequest(respW, req)
-			require.EqualError(t, err, "Secure variable missing required Items object.")
+			require.EqualError(t, err, "secure variable missing required Items object")
```

Comment:
It looks like we missed a few of these: https://app.circleci.com/pipelines/github/hashicorp/nomad/26598/workflows/37b8c8e2-7ffb-4484-a67a-b68082cecb18/jobs/290337